crates/wasi-common/src/old/snapshot_0/sys/windows/hostcalls_impl/fs.rs

```diff
@@ -16,6 +16,7 @@ use std::io::{self, Seek, SeekFrom};
 use std::os::windows::fs::{FileExt, OpenOptionsExt};
 use std::os::windows::prelude::{AsRawHandle, FromRawHandle};
 use std::path::{Path, PathBuf};
+use winapi::shared::winerror;
```

Comment:
Are you sure that you want to modify the snapshot?

Reply:
The changes to the snapshot do not touch either its use of the WASI ABI nor the functional aspects of the code which are identical between both snapshots anyhow, so we're fine here.